http://lists.gnu.org/archive/html/bug-gnulib/2007-05/msg00089.html

2007-05-19  Jim Meyering  <jim@meyering.net>

	* lib/utimens.c (gl_futimens): Rename from futimens,
	now that glibc-2.6 declares futimens.
	* lib/utimens.h: Likewise.

diff --git a/lib/utimens.c b/lib/utimens.c
index 71bc510..7e3175a 100644
--- a/lib/utimens.c
+++ b/lib/utimens.c
@@ -75,8 +75,8 @@ struct utimbuf
    Return 0 on success, -1 (setting errno) on failure.  */
 
 int
-futimens (int fd ATTRIBUTE_UNUSED,
-	  char const *file, struct timespec const timespec[2])
+gl_futimens (int fd ATTRIBUTE_UNUSED,
+	     char const *file, struct timespec const timespec[2])
 {
   /* Some Linux-based NFS clients are buggy, and mishandle time stamps
      of files in NFS file systems in some cases.  We have no
@@ -185,5 +185,5 @@ futimens (int fd ATTRIBUTE_UNUSED,
 int
 utimens (char const *file, struct timespec const timespec[2])
 {
-  return futimens (-1, file, timespec);
+  return gl_futimens (-1, file, timespec);
 }
diff --git a/lib/utimens.h b/lib/utimens.h
index 0097aaa..169521d 100644
--- a/lib/utimens.h
+++ b/lib/utimens.h
@@ -1,3 +1,3 @@
 #include <time.h>
-int futimens (int, char const *, struct timespec const [2]);
+int gl_futimens (int, char const *, struct timespec const [2]);
 int utimens (char const *, struct timespec const [2]);
